### DISCUSSION TITLE
Will the immich app also show local photos?

### DISCUSSION THREAD

**Discussion by skutter-de** [immich-app/immich#27] [General]
I think it should be able to at least. Imagine having a button on the picture view like this: a cloud with a up arrow in it if it's only on the device, a cloud with a check mark if it's on both and a cloud with a down arrow if it's just on the server. While uploading/downloading, the cloud icon should have a progress bar around it.

**Comment by alextran1502** [comment]
My initial prototype was following this approach. I am planning to rewrite the way I manage the total assets. I will think of a way to incorporate this.
